AllReadyApp/Web-App/AllReady/Views/Activity/Activity.cshtml

```diff
@@ -223,30 +223,18 @@
                                         <span asp-validation-for="SignupModel.AdditionalInfo" class="text-danger"></span>
                                     </div>
                                 </div>
-                                @if (Model.RequiredSkills?.Count > 0)
-                                {
-                                    <div class="form-group">
+                                    <div class="form-group" data-bind="visible: unassociatedSkills.length > 0">

```

Comment:
This is kind of silly, but `unassociatedSkills.length > 0` can just be `unassociatedSkills.length` because 0 is "falsy".